README.md

```diff
@@ -127,6 +127,16 @@ npm install -g @pnp/cli-microsoft365
   ```
 </details>
 
+<details>
+  <summary>Run CLI for Microsoft 365 in a Docker container 🐳</summary>
+
+  ```
+  docker run --rm -it m365pnp/cli-microsoft365:latest
+  ```
+
+  Checkout our [guide](user-guide/run-cli-in-docker-container/)) to learn more about how to run CLI for Microsoft 365 using Docker
```

Comment:
Is this link correct since we're in the README.md and it's pointing to our docs?

Reply:
Good catch, yes it is wrong, updated 👍